PROJECT

```diff
@@ -22,4 +22,12 @@ resources:
   group: eraser.sh
   kind: ImageList
   version: v1
+- api:
+    crdVersion: v1
+    namespaced: true
```

Comment:
do we want config to be namespaced?

Reply:
I'm not sure what this is referring to in the PROJECT file. The configmap should be namespaced so the pods can reference it.